modules/twitch_streaming

```diff
@@ -0,0 +1,55 @@
+
+"""
+Checks if a streamer is online. [Gross Gore or grossie_gore as example placeholder]
```

Comment:
I think this could read better. and kill the stuff in square brackets.

Checks if a Twitch.tv streamer is online.